Dockerfile

```diff
@@ -51,3 +51,9 @@ RUN chmod +x /usr/local/bin/swagger
 
 # install misspell
 RUN curl -L https://git.io/misspell | bash
+
+# install yamllint
+RUN apt-get install -y python-pip
```

Comment:
Please move this line to L14, and combine the following 3 lines into a single one.
We can reduce the image size by that.